packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -52,9 +52,13 @@ const GENERATED_ANSWER_DATA_KEY = 'coveo-generated-answer-data';
 
 /**
  * The `QuanticGeneratedAnswer` component automatically generates an answer using Coveo machine learning models to answer the query executed by the user.
+ * This component includes a slot, "no-answer-message", which allows for rendering a custom message when no answer is generated.
  * @category Search
+ * @slot no-answer-message - Slot that allows the rendering of a custom message when no answer is generated.
  * @example
- * <c-quantic-generated-answer engine-id={engineId} with-toggle collapsible></c-quantic-generated-answer>
+ * <c-quantic-generated-answer engine-id={engineId} with-toggle collapsible>
+ *  <div slot="no-answer-message">No answer was generated.</div>
```

Comment:
is it slot= or name= ?

Reply:
it is slot, name is for the <slot> element :P